.eslintrc.js

```diff
@@ -11,5 +11,13 @@ module.exports = {
         'prettier',
         'prettier/@typescript-eslint',
         'plugin:prettier/recommended'
-    ]
+    ],
+    rules: {
```

Comment:
I don't think we want this here. This `prettier/prettier` ESLint plugin is reading the settings from the [prettier.config.js](https://github.com/DataDog/sketches-js/blob/master/prettier.config.js) file, so any formatting-config changes we'd make should go in `prettier.config.js`

I'm guessing that if you ran into an issue with this, instead of swapping the default value for [endOfLine](https://prettier.io/docs/en/options.html#end-of-line), we should set up a pre-commit hook to run `prettier` with the project's configuration (or make sure that your local editor is set up to run `prettier` itself).

Reply:
The problem is that Windows uses CRLF, and by default Prettier enforces LF only. So the build fails on Windows because of this limitation. I can move the rule to `prettier.config.js`, but the rule is still needed.

Reply:
@brimtown When I add a Prettier config file, it looks line the ESLint config file starts to be ignored. From the information I found online, I think when both Prettier and ESLint are combined then any Prettier configuration should be done in the ESLint configuration. Please let me know if you know of a way to use both.